docs/products/kafka/karapace/howto/manage-kafka-rest-proxy-authorization.rst

```diff
@@ -0,0 +1,11 @@
+Manage Apache Kafka® REST proxy authorization
+==============================================
+
+Apache Kafka® REST proxy authorization allows you to use the RESTful interface to connect to Kafka clusters, produce and consume messages easily, and execute administrative activities using Aiven CLI. This feature is disabled by default, and you need to :doc:`enable Apache Kafka REST proxy authorization <../howto/enable-kafka-rest-proxy-authorization>`.
+
+When you enable Apache Kafka REST proxy authorization, Karapace sends the HTTP basic authentication credentials to Apache Kafka®. The authentication and authorization are then performed by Apache Kafka, depending on the ACL defined in Apache Kafka. To configure the ACLs for authorization, see :doc:`Kafka Access Control Lists (ACLs) </docs/products/kafka/concepts/acl>`.
+
+.. note:: 
+    If the feature is **disabled**, Apache Kafka REST proxy authorization connects to Apache Kafka® using a special **superuser** with full access to all topics.
+
```

Comment:
It's actually not exactly like this, this affects which user account the Console uses when producing / consuming to topics, creating/deleting topics. Here the Console calls AIven REST API that calls Kafka REST proxy.

If the feature is enabled:

Aiven platform creates a special superuser called `.kafka_rest_proxy` in Kafka and adds ACL entries to Kafka for that. Aiven REST API calls Kafka REST proxy using this user. This means that when e.g. producing to Kafka in the Console, this user will be used when producing to Kafka.

If the feature is disabled:

The REST Proxy connects to a dedicated listener in Kafka that bypasses the Kafka ACLs, allowing REST Proxy to perform any operations in Kafka. This means that e.g. in Console if you consume or produce to a Kafka topic, the Console effectively calls Kafka REST proxy that calls Kafka. And this call to Kafka bypasses the Kafka ACLs completely.

Reply:
@harshini-rangaswamy, are you going to work on Juha's feeedback?

Reply:
> @harshini-rangaswamy, are you going to work on Juha's feeedback?

Yes, I have pinged Juha for more clarifications and am waiting for input.